react/MinicartContext.tsx

```diff
@@ -7,20 +7,30 @@ interface OpenMinicartAction {
 interface CloseMinicartAction {
   type: 'CLOSE_MINICART'
 }
+interface SetOpenOnHoverBehaviorAction {
+  type: 'SET_OPEN_ON_HOVER_BEHAVIOR'
+  value: boolean
+}
 
 interface State {
   variation: MinicartVariationType
   open: boolean
   hasBeenOpened: boolean
-  openOnHover: boolean
+  /** Whether or not the expected openOnHover behavior is active */
+  openOnHoverBehavior: boolean
```

Comment:
Maybe can be like

`openBehaviour: 'hover' | 'click'`?

Reply:
Nice!